dashboard/app/models/script.rb

```diff
@@ -436,6 +448,51 @@ def self.get_from_cache(id_or_name, version_year: nil)
     end
   end
 
+  def self.get_family_without_cache(family_name)
+    Script.where(family_name: family_name).order("properties -> '$.version_year' DESC")
+  end
+
+  # Returns all scripts within a family from the Rails cache.
+  # Populates the cache with scripts in that family upon cache miss.
+  # @param family_name [String] Family name for the desired scripts.
+  # @return [Array<Script>] Scripts within the specified family.
+  def self.get_family_from_cache(family_name)
+    return Script.get_family_without_cache(family_name) unless should_cache?
+
+    script_family_cache.fetch(family_name) do
+      # Populate cache on miss.
+      script_cache[family_name] = Script.get_family_without_cache(family_name)
```

Comment:
should be `script_family_cache[...] =` ? if not... weird things may happen

Reply:
yes! i was originally using `script_cache` instead of `script_family_cache` and missed a spot. thank you!